jupyter/muehle-game.ipynb

```diff
@@ -1,32 +1,8 @@
 {
```

Comment:
Uff, bitte die Änderungen in `muehle-game.ipynb` rückgängig machen.

Reply:
man man man, da hab ich ja total verkackt. xD 
In dem Commit war nichtmal der eigentliche change, sondern wirklich nur (fälschlicherweise) `muehle-game.ipynb`.

Daher konnte ich einfach den commit 8570ffa wieder reverten 9d9bd7d

Habs kurz getestet, sollte jetzt wieder alles funken :)